weather_mv/loader_pipeline/ee.py

```diff
@@ -294,8 +303,26 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
                             region=pipeline_options_dict.get('region'))
             logger.info('Region validation completed successfully.')
 
+        # Check for the band_names json file
+        if not os.path.exists(known_args.band_names):
+            raise RuntimeError("--band_names should contain a valid file that exists")
+        else:
+            if not known_args.band_names.split('.')[-1] == 'json':
+                raise RuntimeError("--band_names should contain a json file as input")
+
+        # Check the initialization_time and forecast_time strings
+        chars = ['%Y', '%m', '%d', '%H', '%M', '%S']
+        if not all([char in known_args.initialization_time for char in chars]):
+            raise RuntimeError("--initialization_time doesn't contain(s) [%Y, %m, %d, %H, %M, %S]")
```

Comment:
I think you can refer to the variable via a format string like so: 

```
f"--initialization_time doesn't contain(s) {chars!s}"
```

Reply:
Resolved